packages/frontend-2/lib/projects/graphql/queries.ts

```diff
@@ -347,6 +347,30 @@ export const projectWebhooksQuery = graphql(`
   }
 `)
 
+export const projectEmbedTokensQuery = graphql(`
+  query ProjectEmbedTokens($projectId: String!, $cursor: String = null) {
+    project(id: $projectId) {
+      id
+      name
+      ...ProjectPageSettingsTokens_Project
+      embedTokens(cursor: $cursor, limit: 20) {
+        cursor
+        totalCount
+        items {
+          createdAt
```

Comment:
this seems like it should be a fragment belonging to the page where these props are actually used